pubsub/google/cloud/pubsub_v1/publisher/exceptions.py

```diff
@@ -11,3 +11,10 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+class PublishError(RuntimeError):
```

Comment:
Do either of these bubble up to users? If yes, they should inherit from `google.cloud.exceptions.GoogleCloudError`.